tutorindigo/plugin.py

```diff
@@ -102,68 +105,27 @@ def _override_openedx_docker_image(
 hooks.Filters.CONFIG_OVERRIDES.add_items(list(config["overrides"].items()))
 
 
+#  MFEs that are styled using Indigo
+indigo_styled_mfes = ['learning', 'learner-dashboard', 'profile', 'account', 'discussions']
+
 hooks.Filters.ENV_PATCHES.add_items(
-    [
-        # MFE will install header version 3.0.x and will include indigo-footer as a
-        # separate package for use in env.config.jsx
-        (
-            "mfe-dockerfile-post-npm-install-learning",
+    [(
+        f"mfe-dockerfile-post-npm-install-{mfe}",
             """
-RUN npm install '@edx/brand@npm:@edly-io/indigo-brand-openedx@^2.1.1'
-RUN npm install '@edx/frontend-component-header@npm:@edly-io/indigo-frontend-component-header@^3.1.3'
+            
 RUN npm install @edly-io/indigo-frontend-component-footer@^2.0.0
-
-COPY indigo/env.config.jsx /openedx/app/
-""",
-        ),
-        (
-            "mfe-dockerfile-post-npm-install-authn",
-            """
-RUN npm install '@edx/brand@npm:@edly-io/indigo-brand-openedx@^2.1.1'
-""",
-        ),
-        # Tutor-Indigo v2.1 targets the styling updates in discussions and learner-dashboard MFE
-        # brand-openedx is related to styling updates while others are for header and footer updates
-        (
-            "mfe-dockerfile-post-npm-install-discussions",
-            """
-RUN npm install '@edx/brand@npm:@edly-io/indigo-brand-openedx@^2.1.1'
 RUN npm install '@edx/frontend-component-header@npm:@edly-io/indigo-frontend-component-header@^3.1.3'
-RUN npm install @edly-io/indigo-frontend-component-footer@^2.0.0
-
-COPY indigo/env.config.jsx /openedx/app/
```

Comment:
Since we are no longer copying env.config.jsx, do we still need the file in the repo?

Reply:
Thanks for pointing out. I'll fix it.